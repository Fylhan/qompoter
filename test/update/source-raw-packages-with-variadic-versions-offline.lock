{
  "last": "c820f92e2426bcad0a9b6798b2309906",
  "name": "update/source-raw-packages-with-variadic-versions-offline",
  "date": "2021-02-24T15:46:31+01:00",
  "require": {
    "cp/fylhan": { "version": "v1.1.10", "type": "qompotist-fs", "url": "qompoter-repo/cp/fylhan/v1.1.10", "md5sum": "9d57702586a06dcd5de4d717d936c9d6" }
    ,"cp/acme-cp": { "version": "v1.0-lib", "type": "qompotist-fs", "url": "qompoter-repo/cp/acme-cp/v1.0-lib", "md5sum": "115f3cd6f4a5dec31fb31b9cf39fa855" }
    ,"git/acme-git": { "version": "v2.0.2", "type": "git", "url": "qompoter-repo/git/acme-git/acme-git.git", "md5sum": "c820f92e2426bcad0a9b6798b2309906" }
  }
}
