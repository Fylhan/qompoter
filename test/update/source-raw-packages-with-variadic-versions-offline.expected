Qompoter
======== install

* cp/fylhan v1.1.10
  Downloading sources...
  done

* cp/acme-cp v1.0-lib
  Downloading lib...
  done

* git/acme-git v2.0.2
  Downloading sources from Git...
  Hint: Think about upgrading to the last version v2.0.3
  done

done
